src/extension.ts

```diff
@@ -67,6 +67,12 @@ export function activate(context: vscode.ExtensionContext) {
   });
   context.subscriptions.push(installCompiler);
 
+  // Target Device view
+  let connectDevice = vscode.commands.registerCommand('onevscode.connect-device', () => {
```

Comment:
Can I ask a question about the naming `connect`?

Is it a command when, e.g, `$ adb connect` is called?

While a user see a target device in a Target Device View,
internally the device can be connected and disconnected.

So I wonder this command is for such connection.

Reply:
> Is it a command when, e.g, `$ adb connect` is called?

I think it will call `$ adb connect` once to check device is connectable.

> While a user see a target device in a Target Device View,
> internally the device can be connected and disconnected.

but, this case would be right. so I think `register` would be more right for this.
`Register` this device as a `Target Device` on user or one-vscode configuration.

so, what about change this into `register`?